<launch>
    <node name="traffic_supervisor" pkg="smartgv_realsense" type="traffic_supervisor.py" output="screen"></node>
</launch>